### PR TITLE
Remove unnecessary pass-by-ref

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Remove unnecessary pass-by-ref

Before
----------------------------------------
image

After
----------------------------------------
no more

Technical Details
----------------------------------------
Also stopped passing in a pass-by-ref variable in the function called from these places

image

Comments
----------------------------------------
Copy *& paste strikes again...

### CODE REVIEW

File: CRM/Import/Form/DataSource.php [hunk 1]
```diff
@@ -150,7 +150,7 @@ protected function submitFileForMapping($parserClassName, $entity = NULL) {
     $parser->setMaxLinesToProcess(100);
     $parser->run($fileName,
       $separator,
-      $mapper,
+      [],
```
Comment (colemanw) [inline CRM/Import/Form/DataSource.php:153]: Looks like you could also delete the now-unused variable `$mapper`

Comment (eileenmcnaughton) [reply CRM/Import/Form/DataSource.php:153]: @colemanw not quite yet 

image

**Comment by eileenmcnaughton** [comment]
rebased

**Comment by colemanw** [comment]
Ok.

**Pull request merged**